mne/channels/channels.py

```diff
@@ -975,6 +975,9 @@ def interpolate_bads(self, reset_bads=True, mode='accurate',
             Either ``'accurate'`` or ``'fast'``, determines the quality of the
             Legendre polynomial expansion used for interpolation of MEG
             channels.
+        fnirs_method : str
+            Method to be used for fNIRS interpolation. Currently only 'nearest'
+            is supported.
```

Comment:
I would not expose this parameter at this stage. Interpolation is done differently for each channel type now.